net/miniupnpc/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpc
-PKG_VERSION:=1.9
+PKG_VERSION:=2.0
```

Comment:
Please bump it to the latest snapshot while at it.
miniupnpc-2.0.20161216.tar.gz at the time of writing